lightningd/channel_control.c

```diff
@@ -412,6 +411,43 @@ static void handle_funding_tx(struct channel *channel,
 			   sendfunding_done, cs);
 }
 
+static void peer_tx_sigs_msg(struct channel *channel, const u8 *msg)
+{
+	struct wally_psbt *psbt;
+	const struct wally_tx *wtx;
+	struct lightningd *ld = channel->peer->ld;
+
+	if (!fromwire_channeld_funding_sigs(tmpctx, msg, &psbt)) {
+		channel_internal_error(channel,
+				       "bad channeld_funding_sigs: %s",
+				       tal_hex(tmpctx, msg));
+		return;
+	}
+
+	tal_wally_start();
+	if (wally_psbt_combine(cast_const(struct wally_psbt *,
+					  channel->psbt),
```

Comment:
This breaks the rules; channel->psbt must not be const if we want to alter it.  Casting away just removes the warning, but it's still no legal to alter it.

Reply:
why is const. so unhappy. all the time.